lib/active_model/serializer/adapter/json.rb

```diff
@@ -9,12 +9,6 @@ def serializable_hash(options = nil)
           options ||= {}
           { root => Attributes.new(serializer).serializable_hash(options) }
         end
-
-        private
-
-        def fragment_cache(cached_hash, non_cached_hash)
-          ActiveModel::Serializer::Adapter::Json::FragmentCache.new.fragment_cache(cached_hash, non_cached_hash)
-        end
```

Comment:
I think this method still needs to be here, but I don't totally grok the caching code yet cc @joaomdmoura